src/main/java/org/craftercms/studio/impl/v2/service/configuration/ConfigurationServiceImpl.java

```diff
@@ -188,6 +190,12 @@ public Document getConfigurationAsDocument(@ProtectedResourceId(SITE_ID_RESOURCE
         return retDocument;
     }
 
+    @Override
+    @HasPermission(type = DefaultPermission.class, action = "write_global_configuration")
```

Comment:
Same comment as before, shouldn't we add both `read` & `write` permission?

Reply:
We'll proceed with `write` implies `read` and for this tool, you need `write` to do anything meaningful.